starport-core/src/main/scala/com/krux/starport/db/table/ScheduledPipelines.scala

```diff
@@ -55,8 +55,19 @@ class ScheduledPipelines(tag: Tag) extends Table[ScheduledPipeline](tag, "schedu
    * If this still shows in the AWS Data Pipeline console
    */
   def inConsole = column[Boolean]("in_console")
+  /**
+   * The final pipeline status for monitoring purpose
+   */
```

Comment:
no new line after, but a new line before this comment

Reply:
updated the formatting as requested.